### PR TITLE
Feature/Add ability to build Stacked arcs for StackedRadialGaugeSeries

### PR DESCRIPTION
**Pull request by SerhiiTsybulskyi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
StackedRadialGaugeSeries doesn't have ability to build stacked arcs

Issue Number: #166 

## What is the new behavior?
RadialGauge have ability to process data with nested structure (ChartDataShape) and draw stacked arcs for nested data

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

https://github.com/reaviz/reaviz/assets/5399389/759b4f97-6332-462f-abad-fc999b4b6873

### CODE REVIEW

**Comment by evgenoid** [comment]
@amcdnl when we going to publish it?

File: src/RadialGauge/RadialGauge.story.tsx [hunk 1]
```diff
@@ -126,6 +129,119 @@ export const Stacked = () => {
...
+  }
+
+  const descriptionElement = (
+    <text
+      x="0"
+      y={-20}
+      textAnchor="middle"
+      alignmentBaseline="middle"
+    >
+      <tspan x="0" fontSize={16} fontWeight={700} fill={'#E9E9E9'}>
```
Comment (steppy452) [inline src/RadialGauge/RadialGauge.story.tsx:186]: I don't think these `fill` need to be backticks - it's just a string literal

File: src/RadialGauge/RadialGauge.story.tsx [hunk 2]
```diff
@@ -126,6 +129,119 @@ export const Stacked = () => {
...
 
+interface GaugeStackedTemplateProps {
+  width: number;
+  height: number;
+  data: ChartDataShape[];
+  maxValue?: number;
+  colorSchemaType: 'handlerFn' | 'array' | 'string';
+}
+
+const GaugeStackedTemplate: StoryFn<GaugeStackedTemplateProps> = (args) => {
```
Comment (steppy452) [inline src/RadialGauge/RadialGauge.story.tsx:140]: prefer `props` instead of `args` just for consistency but even better, prefer destructuring

**Review by evgenoid — APPROVED** [review]

File: src/RadialGauge/RadialGaugeSeries/StackedRadialGaugeSeries.tsx [hunk 3]
```diff
@@ -160,6 +230,7 @@ export const StackedRadialGaugeSeries: FC<
 StackedRadialGaugeSeries.defaultProps = {
   outerArc: <RadialGaugeArc disabled={true} animated={false} />,
   innerArc: <RadialGaugeArc animated={true} />,
+  stackedInnerArc: <RadialGaugeStackedArc animated={true} />,
```
Comment (amcdnl) [inline src/RadialGauge/RadialGaugeSeries/StackedRadialGaugeSeries.tsx:233]: Should we turn this off by default ( use null ) ?

**Review by amcdnl — COMMENTED** [review]
One minor ? but otherwise lgtm

**Pull request merged**